docs/variants/dell_optiplex/installation-manual.md

```diff
@@ -59,18 +59,46 @@ active. Press F1 to proceed and boot to your Linux system.
 
     ![](../../images/service_mode_warn.jpg)
 
+## Install flashrom
+
+* Install flashrom v1.1 or newer with your distribution's package manager if
+  you don't have it installed yet. If your distro doesn't provide flashrom or
+  provides an outdated one, you can build it yourself using
+  [this instruction](https://www.flashrom.org/Downloads).
+* Or compile recent version of flashrom:
+  ``` console
+  sudo apt install libpci-dev libftdi-dev libusb-1.0-0-dev
+  git clone https://github.com/flashrom/flashrom.git
+  cd flashrom
+  sudo make install
+  ```
+
+## BIOS backup
+
+<!-- BIOS backup section is very generic and should be treated in such way This
+section even can be replaced with Dasharo Reference OS, fwupd or other tools
+that can simplify the operation for the user -->
+
+It is always good idea to backup original BIOS of your hardware, before
+switching to open source firmware.
+
+* Read content of SPI NOR flash:
+  ``` console
+  sudo flashrom -p internal -r bios_backup_`date +%Y%m%d`.bin
+  ```
+
```

Comment:
Do we want to include flashrom logs indicating the flash descriptor security override pin strap is set?

Reply:
Hmm, I didn't saw anything special but maybe I missed something. Let me check.

Reply:
Flashrom should report the strap pin is set

Reply:
@miczyg1 added flashrom log. Please check. If it looks ok and no other issues, please merge.

Reply:
Looks good.